hack/pre-release.sh

```diff
@@ -187,9 +190,19 @@ switch_git_branch $base_branch $new_branch
 WMCO_VERSION=$(sed -n -e 's/^.*WMCO_VERSION ?= //p' Makefile)
 update_WMCO_version $stream $WMCO_VERSION
 update_OCP_version $stream $WMCO_VERSION 
+
+# check to see if the opeator-sdk version is correct 
+operator_sdk_current_version=$(operator-sdk version |  grep -oP 'operator-sdk version: "\K[^"]+' )
+if [[ "$operator_sdk_current_version" != "$OPERATOR_SDK_VERSION" ]]; then
+  echo "operator-sdk $OPERATOR_SDK_VERSION must be installed."  
+  echo "exiting."
+  git checkout $initial_branch
+  exit 
+fi
+
 make bundle
 
-commit_message="[release] Update $base_branch $stream stream to version
+commit_message="[release] Update $base_branch $stream stream to $WMCO_VERSION 
```

Comment:
```suggestion
commit_message="[$base_branch] Update version to $WMCO_VERSION`
```
Including the stream in the commit message when you mentioning the full version is a bit odd.

Reply:
This has the added advantage of having the branch prefix when you the open the version bump PR.